config/ember-try.js

```diff
@@ -1,5 +1,189 @@
 /* eslint-env node */
 module.exports = {
   command: 'yarn test',
-  useVersionCompatibility: true
+  scenarios: [
+    {
+      name: 'default',
+      dependencies: { }
+    },
+    {
+      name: 'ember-1.12',
+      bower: {
+        dependencies: {
+          ember: '~1.12.0',
+          'ember-cli-shims': '0.0.6',
+          'ember-data': '~1.13.0',
```

Comment:
shouldn't this use Ember Data 1.12 too?

Reply:
Ember Data 1.12 doesn't seem to exist…

Reply:
ah, indeed. according to https://github.com/emberjs/data/tags?after=v1.13.3 there was only v1.0.0 betas before the v1.13.0 release 🤔

Reply:
The Ember Data version doesn't really matter much for the ESA tests though. In fact it's only needed so that the current user can be fetched during an acceptance test where the fetching itself doesn't really relate to ESA at all.

Reply:
> there was only v1.0.0 betas before the v1.13.0 release 🤔

Those were the wild and crazy days, in particular with Ember Data after all 😜